actionpack/test/dispatch/actionable_exceptions_test.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "abstract_unit"
+
+class ActionableExceptionsTest < ActionDispatch::IntegrationTest
+  Actions = []
+
+  class ActionError < StandardError
+    include ActiveSupport::ActionableError
+
+    action "Successful action" do
+      Actions << "Action!"
+    end
+
+    action "Failed action" do
+      fail "Inaction!"
```

Comment:
Nitpick: You should use `raise` instead of `fail`, same in the assertion string.